### PR TITLE
Setup controller

### PR DESCRIPTION
**Pull request by Tumworobere**
**My-Blog-App: Setup and Controllers**

   - Created a new app. ✔️
   - Initialized my project with Git. ✔️
   - Postgres database has been set up ✔️
   - Added code to handle only the 4 required URLs.✔
   -  View files should include only placeholders written in plain HTML ✔

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  get 'user/:user_id/posts' => 'posts#index', as: 'user_posts'
+  get 'users/:user_id/posts/:id' => 'posts#show', as: 'user_post'
+  get 'users' => 'users#index', as: 'users'
+  get 'users/:id' => 'users#show', as: 'user'
```
Comment (rukundoeric) [inline config/routes.rb]: - [ ] <b>REQUIRED:</b> Here is more preferred way of setting up routes, Kindly do that to comply with project requirements.
```rb
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Tumworobere, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Tumworobere** [comment]
> Hi @Tumworobere,
> 
> _Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
> ## Highlights
> 
>     * Follows git workflow.  💯
> 
>     * Codebase looks clean and professional. 💯
> 
> 
> ## Required Changes ♻️
> 
> _Check the comments under the review._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> > As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Thank you @rukundoeric .I will work on the changes

**Review by DeliceLydia — APPROVED** [review]
## Hi @Tumworobere, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tumworobere** [comment]
> ## Hi @Tumworobere,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> # Congratulations! 🎉
> ### Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you @DeliceLydia

**Pull request merged**